.gitignore

```diff
@@ -1 +1,2 @@
-vendor
\ No newline at end of file
+vendor
+.envrc
```

Comment:
Sorry for nitpicking, but it's better to ignore `.envrc` through a global gitignore. This way, only files actually generated by the repository will end and in the repository's `.gitignore` file.